features/networking/service.feature

```diff
@@ -633,3 +633,82 @@ Feature: Service related networking scenarios
       | KUBE-SEP-.+ -p tcp .* -m tcp -j DNAT --to-destination <%= cb.pod_ip %>:8080            |
       | KUBE-SERVICES -d <%= cb.service_ip %>/32 -p tcp .* -m tcp --dport 27017 -j KUBE-SVC-.+ |
       | KUBE-SVC-.+ .* -j KUBE-SEP-.+                                                          |
+
+
+  # @author jechen@redhat.com
+  # @case_id OCP-43493
+  @admin
+  @destructive
+  @4.10 @4.9
+  @vsphere-ipi @openstack-ipi @gcp-ipi @baremetal-ipi @azure-ipi @aws-ipi
+  @vsphere-upi @openstack-upi @gcp-upi @azure-upi @aws-upi
+  Scenario: Update externalIP from oc edit svc
+    Given I have a project
+    And evaluation of `project.name` is stored in the :proj_name clipboard 
+    And SCC "privileged" is added to the "system:serviceaccounts:<%= project.name %>" group
+    Given I store the schedulable nodes in the :nodes clipboard
+
+    # Add externalIP policy to CNO
+    Given as admin I successfully merge patch resource "networks.config.openshift.io/cluster" with:
+      | {"spec":{"externalIP":{"policy":{"allowedCIDRs":["1.1.1.0/24"]}}}} |
+
+    # Clean-up required to erase above externalIP policy after testing done
+    Given I register clean-up steps:
+    """
+    Given as admin I successfully merge patch resource "networks.config.openshift.io/cluster" with:
+      | {"spec":{"externalIP":{"policy":{"allowedCIDRs": null}}}} |
+    """
+
+    # Create a svc with externalIP
+    Given I switch to the first user
+    And I use the "<%= cb.proj_name %>" project
+    Given I obtain test data file "networking/externalip_service1.json"
+    And I wait up to 300 seconds for the steps to pass:
+    """
+    When I run oc create over "externalip_service1.json" replacing paths:
+      | ["spec"]["externalIPs"][0] | 1.1.1.1 |
+    Then the step should succeed
+    """
+
+    # Create a pod
+    Given I obtain test data file "networking/externalip_pod.yaml"
+    When I run the :create client command with:
+      | f | externalip_pod.yaml |
+    Then the step should succeed
+    And the pod named "externalip-pod" becomes ready
+
+    # Curl externalIP:portnumber should pass
+    Given I switch to cluster admin pseudo user
+    And I use the "<%= cb.proj_name %>" project
+    When I run the :debug admin command with:
+      | resource     | node/<%= cb.nodes[0].name %> |
+      | oc_opts_end  |                              |
+      | exec_command | chroot                       |
+      | exec_command | /host                        |
+      | exec_command | curl                         |
+      | exec_command | 1.1.1.1:27017                |
+    Then the step should succeed
```

Comment:
we can use 
```
    Given I use the "<%= cb.nodes[0].name %>" node
    And I run commands on the host:
      | curl 1.1.1.1:27017 |
```

Reply:
Thanks, I have used this method.  But for next step of patch changing externalIP for svc, I had to add the following to re-enforce the "oc patch" command is executed with right project.  I found after curil 1.1.1.1:27017 using the method you suggested, it has left my project.
     Given I switch to cluster admin pseudo user
     And I use the "<%= cb.proj_name %>" project

Reply:
sorry, I forgot to mentioned the `--connect-timeout 5` when `curl`